<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calFilterPropForm</class>
 <widget class="QWidget" name="calFilterPropForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Result Array Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resultName_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="cal_lineEdit"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="clear_pushButton">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="left_pushButton">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="right_pushButton">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="iHat_pushButton">
       <property name="text">
        <string>iHat</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="jHat_pushButton">
       <property name="text">
        <string>jHat</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="kHat_pushButton">
       <property name="text">
        <string>kHat</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="sin_pushButton">
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="cos_pushButton">
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="tan_pushButton">
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="abs_pushButton">
       <property name="text">
        <string>fabs</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="sqrt_pushButton">
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="add_pushButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="asin_pushButton">
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="acos_pushButton">
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="atan_pushButton">
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="ceil_pushButton">
       <property name="text">
        <string>ceil</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="floor_pushButton">
       <property name="text">
        <string>floor</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="sub_pushButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="sinh_pushButton">
       <property name="text">
        <string>sinh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="cosh_pushButton">
       <property name="text">
        <string>cosh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="tanh_pushButton">
       <property name="text">
        <string>tanh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="sqrtxy_pushButton">
       <property name="text">
        <string>x^y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="exp_pushButton">
       <property name="text">
        <string>exp</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="mul_pushButton">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="v1v2_pushButton">
       <property name="text">
        <string>v1.v2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="mag_pushButton">
       <property name="text">
        <string>mag</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="norm_pushButton">
       <property name="text">
        <string>norm</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="ln_pushButton">
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="log10_pushButton">
       <property name="text">
        <string>log10</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="div_pushButton">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="scalars_comboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="vectors_comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>238</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
